sources/web/datalab/server.ts

```diff
@@ -228,6 +228,16 @@ function handleRequest(request: http.ServerRequest,
 }
 
 /**
+ * Returns true iff the supplied path should be handled by the static handler
+ */
+function isStaticResource(urlpath: string) {
```

Comment:
It would make a bit more sense to put this in static.ts. Perhaps in the next PR.